airflow/providers/google/cloud/operators/stackdriver.py

```diff
@@ -0,0 +1,743 @@
+        For more details, see `sorting and filtering
+        <https://cloud.google.com/monitoring/api/v3/sorting-and-filtering>`__.
+    :type filter_: str
+    :param order_by: A comma-separated list of fields by which to sort the result.
+        Supports the same set of field references as the ``filter`` field. Entries
+        can be prefixed with a minus sign to sort by the field in descending order.
+        For more details, see `sorting and filtering
+        <https://cloud.google.com/monitoring/api/v3/sorting-and-filtering>`__.
+    :type order_by: str
+    :param page_size: The maximum number of resources contained in the
+        underlying API response. If page streaming is performed per-
+        resource, this parameter does not affect the return value. If page
+        streaming is performed per-page, this determines the maximum number
+        of resources in a page.
+    :type page_size: int
+    :param retry: A retry object used to retry requests. If ``None`` is
+        specified, requests will be retried using a default configuration.
+    :type retry: str
+    :param timeout: The amount of time, in seconds, to wait
+        for the request to complete. Note that if ``retry`` is
+        specified, the timeout applies to each individual attempt.
+    :type timeout: float
+    :param metadata: Additional metadata that is provided to the method.
+    :type metadata: str
+    :param gcp_conn_id: (Optional) The connection ID used to connect to Google
+        Cloud Platform.
+    :type gcp_conn_id: str
+    :param project_id: The project to fetch alerts from.
+    :type project_id: str
+    :param delegate_to: (Optional) The account to impersonate, if any.
+        For this to work, the service account making the request must have
+        domain-wide delegation enabled.
+    :type delegate_to: str
+    """
+
+    template_fields = ('filter_',)
+    ui_color = "#e5ffcc"
+
+    # pylint: disable=too-many-arguments
+    @apply_defaults
+    def __init__(
+        self,
+        format_=None,
+        filter_=None,
+        order_by=None,
+        page_size=None,
+        retry=DEFAULT,
+        timeout=DEFAULT,
+        metadata=None,
+        gcp_conn_id='google_cloud_default',
+        project_id=None,
+        delegate_to=None,
+        *args, **kwargs
+    ):
+        super().__init__(*args, **kwargs)
+        self.format_ = format_
+        self.filter_ = filter_
+        self.order_by = order_by
+        self.page_size = page_size
+        self.retry = retry
+        self.timeout = timeout
+        self.metadata = metadata
+        self.gcp_conn_id = gcp_conn_id
+        self.project_id = project_id
+        self.delegate_to = delegate_to
+        self.hook = None
+
+    def execute(self, context):
+        if self.hook is None:
+            self.hook = StackdriverHook(gcp_conn_id=self.gcp_conn_id, delegate_to=self.delegate_to)
+
+        return self.hook.list_alert_policies(
```

Comment:
Would it be worth to add some info logging before and after hook call?

Reply:
please check https://github.com/apache/airflow/pull/7322/commits/86898011b3a31e03830d56af567193a6e6d54a2c